index.js

```diff
@@ -43,7 +44,11 @@ function elbMetrics(startTime, endTime, region, elbname, callback) {
     };
 
     var queries = prepareQueries(parameters);
-    outputMetrics(queries, region, callback);
+    outputMetrics(queries, region, function (err, data) {
+        if (err) return callback(err);
+        var requestPercentages = prepareResults(startTime, endTime, data);
+        return callback(null, requestPercentages);
```

Comment:
[style only] You don't need to explicitly `return` the callback here, since it's the last thing that happens.